setup.py

```diff
@@ -4,7 +4,7 @@
 # version checking derived from https://github.com/levlaz/circleci.py/blob/master/setup.py
 from setuptools.command.install import install
 
-VERSION = '0.10.0'
+VERSION = '0.11.0-t2test'
```

Comment:
Could you revert the above line?
Our current process is to update the version number and update the CHANGELOG in a commit separate from the actual changes. For example see:
https://github.com/Duke-GCB/calrissian/commit/8b32f34332220832dbc19f55c09e32fd36d94277
That way we can review all the changes across multiple PRs when considering how to update the version number.